packages/editor/src/store/effects/test/reusable-blocks.js

```diff
@@ -37,6 +36,7 @@ import {
 	fetchReusableBlocks as fetchReusableBlocksAction,
 } from '../../actions';
 import reducer from '../../reducer';
+import '../../..';
```

Comment:
Can we add a comment why we import this thing? It can be copy and paste from what we had before.

Reply:
> Can we add a comment why we import this thing? It can be copy and paste from what we had before.

Updated in rebased 009b8e916b3ae9fac32af65261713feab454097f.